src/com/Jacksonnn/DCCore/DCPlayerListener.java

```diff
@@ -49,7 +49,7 @@ public void onLeave(PlayerQuitEvent e) {
             Bukkit.getLogger().info("[DCCore] There is not a DCPlayer for " + e.getPlayer().getName() + " (" + e.getPlayer().getUniqueId() + ")... Error code: 4.");
         }
     }
-
+ 
     @EventHandler
```

Comment:
What does this look for on death

Reply:
@Jacksonnnnn What does this look for on death

Reply:
@SaltyGraham It keeps the death and kill statistics, so on any death, it finds out if you were killed or the killer and increases the kill/death count.